### ISSUE TITLE
[harm_unit_loti] doesn't recognize first and second unit

### ISSUE THREAD

**Issue by Toranks** [Dugy/Legend_of_the_Invincibles#487]
To check, load this game and attack the pirate with Akula. The savegame is modified to have a 100% chance to disintegrate.
You will see that the pirate talks to herself as if she were the attacker and the victim at the same time.

[LotI1-Piratas Turno 18 harm_unit_loti.gz](https://github.com/Dugy/Legend_of_the_Invincibles/files/11672259/LotI1-Piratas.Turno.18.harm_unit_loti.gz)

Now load this other savegame, here in addition to 100% chance, harm_unit_loti was changed to the original harm_unit. As you can see, the pirate now talks correctly with Akula.

[LotI1-Piratas Turno 18 harm_unit.gz](https://github.com/Dugy/Legend_of_the_Invincibles/files/11672280/LotI1-Piratas.Turno.18.harm_unit.gz)

**Comment by Discontinuum** [comment]
I didn't check the replays, I think this may be the case (vaguely remember something) but can't say for sure. But it's very strange if it's the case, the comment in lua said the lua implementation of harm_unit_loti was supposed to be a copypaste of harm_unit with some tweaks for experience in aoe attacks. In theory it should work the same.

**Comment by Dugy** [comment]
Yes, but a lot of things have changed in the original since the copypaste was made, and it's possible something in the lua engine has changed and the lua code of harm_unit was updated, while harm_unit_loti wasn't.
